src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue7432.cs

```diff
@@ -0,0 +1,19 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue7432 : _IssuesUITest
+{
+    public Issue7432(TestDevice device) : base(device) { }
+
+    public override string Issue => "Android Image.Scale produces wrong layout";
+
+    [Test]
+    [Category(UITestCategories.Image)]
+    public void ImageShouldScaleProperly()
+    {
+        App.WaitForElement("Image");
+        VerifyScreenshot();
```

Comment:
Pending snapshot for iOS, Mac and Windows.

Reply:
Once the CI is triggered, we can add the pending Mac and Windows snapshots @jsuarezruiz